platform/core/src/services/HangingProtocolService/custom-attribute/isDisplaySetFromUrl.ts

```diff
@@ -1,4 +1,4 @@
-import { getSplitParam } from '../../utils';
+import { getSplitParam } from '../../../utils';
```

Comment:
For whatever reason I cannot make comments on any other line other than this first line.

So on line 34, consider using `const displaySet = displaySets ?? displaySets[0]`;

Reply:
The syntax is
const displaySet = displaySets?.[0]
for a condition dereference - the ?? would mean take the display sets list if it is non null.
Applied that change.